docs/workforce-management-framework/topics/con-workforce-management.adoc

```diff
@@ -0,0 +1,35 @@
+[id='{context}-con-wfm-framework-concepts']
+= Introducing Workforce Management
+
+Workforce Management is the management of resources applied outside of the boundaries and property of the company or enterprise itself, including employees, vehicles, consumables and all other property.
+
+Tasks facilitated by Worforce Management software consist of schedule and assignment of work orders to workers, inventory control, customer acceptance of results, quality control, productivity tracking, regulatory compliance, billing, and including others.
+
+== Workforce Management Components in {PROJECT_NAME}
+
+{PROJECT_NAME} introduces the following components to compose a complete Workforce Management solution:
+
+Step::
+A _Step_ is a packaged piece of functionality that usually represents a single unit of work to be done by an entity (usualy a human worker), for example, functionality that allows the worker to capture a customer's signature, a specialized form to be filled in the field, etc.
++
+_Step Definitions_ are usually implemented by a developer with the intent of reusability in multiple _Work Flows_.
+
+Work Flow::
+A _Work Flow_ can contain one to many _Step Definitions_. It defines the order the _Step Definitions_ are executed and are exposed to administrator users as templates to define new Work Orders.
++
+_Work Flows_ are typically handled by administrator users in the Portal Application to create new _Work Orders_ for assignment to workers.
+
+Work Order::
```

Comment:
Reminder to team to correct the names on entire document. I see this format for the first time.

Reply:
That's something I'm still not 100% sure which version we should settle on. We're used to using versions like `workorder` in code, etc. But I think this is the only right way from a gramatical pov :stuck_out_tongue: